test/system/sessions_test.rb

```diff
@@ -17,23 +17,27 @@ class SessionsTest < ApplicationSystemTestCase
 
     assert page.has_field? 'Email'
 
-    fill_in 'Email', with: 'superadmin@example.com'
+    fill_in 'Email', with: @superadmin.email
     fill_in 'Password', with: 'asdfasdf'
-    click_button 'Log in'
-
+    click_on 'Log in'
     assert page.has_current_path? root_path
     assert page.has_text? 'Signed in successfully.'
-    assert page.has_link? 'superadmin@example.com'
+
+    click_on 'Toggle navigation' if page.has_button? 'Toggle navigation'
```

Comment:
We should get rid of this if, we control the dimensions of the window so we never should have the need to use something like this
Tests in general should be as stupid as possible